functions/packages.bash

```diff
@@ -117,8 +118,8 @@ exim_setup() {
   fi
   if cond_redirect install_dnsutils; then echo "OK"; else echo "FAILED"; return 1; fi
 
-  interfaces="$(dig +short "$HOSTNAME" | tr '\n' ';')127.0.0.1;::1"
-  relaynets="$(dig +short "$HOSTNAME" | cut -d'.' -f1-3).0/24"
+  interfaces="$(dig +short "$hostname" | tr '\n' ';')127.0.0.1;::1"
+  relaynets="$(dig +short "$hostname" | cut -d'.' -f1-3).0/24"
```

Comment:
Same question as earlier, does this work if the hostnames are not the same?

Reply:
this didn't change so assuming it worked before it still will

Reply:
It did change though you changed it from $HOSTNAME to $hostname.